setup.py

```diff
@@ -0,0 +1,76 @@
+import setuptools
+from setuptools.command.install import install
+from setuptools.command.test import test
+from setuptools import setup, Command
+import subprocess
+import os
+import unittest
+
+BASEPATH = os.path.dirname(os.path.abspath(__file__))
+NFSV4_PATH = os.path.join(BASEPATH, 'nfsv4')
+SFS_PATH = os.path.join(BASEPATH, 'sfs')
+
+class SfsInstall(install):
+    def run(self):
+        subprocess.call('make', cwd = NFSV4_PATH)
+        install.run(self)
+
+class SfsTest(test):
+    description = "run sfs_test.py using unittest"
+    user_options = [
+        ('ip=', None, 'IP of NFS4_SERVER.'),
+    ]
+    def initialize_options(self):
+        self.ip = None
+    
+    def finalize_options(self):
+        pass
+    
+    def run(self):
+        test_loader = unittest.TestLoader()
+        test_suite = test_loader.discover('test', pattern='sfs_test.py')
+        
+        import sfs
+        try:
+            sfs.mount(self.ip)
+        except Exception as e:
+            print("NFS SSERVER IP: ", self.ip)
+            print(e)
+
+        unittest.TextTestRunner().run(test_suite)
+
+
+class SfsClean(Command):
+    """
+    sfs custom command to clean out junk files.
+    """
+    description = "Cleans out junk files we don't want in the repo"
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        subprocess.call(['make', 'clean'], cwd = NFSV4_PATH)
+        subprocess.call(['rm', 'libnfs4.so'], cwd = SFS_PATH)
+        os.system('rm -vrf ./build ./dist ./*.pyc ./*.tgz ./*.egg-info')
```

Comment:
We have some checked-in files in `.dist`. Does `setup.py` put anything there? If not then we don't need to clean up there. If so we should move away what is already in dist (an old example that probably shouldn't be checked in) to another path.

Reply:
setup.py will only generate build and .egg-info. So running clean should only remove build and .eff-info right now.